lxd/daemon.go

```diff
@@ -746,6 +750,11 @@ func (d *Daemon) Init() error {
 }
 
 func (d *Daemon) setupLoki(URL string, cert string, key string, caCert string, labels []string, logLevel string, types []string) error {
+	// Always cancel the OVN log monitor.
```

Comment:
comment could do with a *why* rather than just a *what*, its not immediately obvious here why ovn log monitor is being cancelled during loki setup.